### DISCUSSION TITLE
Actors?

### DISCUSSION THREAD

**Discussion by duncannevin** [marblejs/marble#382] [Ideas]
Hello!

I was curious if there's any plan to implement actors as part of marble.js? If not I'd be glad to take this one!

Thank you,
Duncan Nevin

**Comment by JozefFlakus** [comment]
Hi! No plans yet, but you are welcome to propose a feature. Waiting for RFC! 🙌

**Comment by duncannevin** [comment]
Very cool, I'll get after it!

[Comedy.js](https://github.com/untu/comedy) has a solid actor framework already so I plan to use their lib. I'll start by building a version with observables.

Reactive streams + Actors + Typescript...whoda thought!

**Reply by JozefFlakus** [reply]
Cool. Please remember to start with RFC! 🙌
